### ISSUE TITLE
[1.18.2] Temperature support is broken for Cold Sweat

### ISSUE THREAD

**Issue by Endgineer** [EtherealOnyx/Parties#167]
`sedparties-1.18.2-1.0.1` used with `ColdSweat-2.2-b05c`, getting the following chat message:
```
[Parties] Cold Sweat needs to be version 2.2 or higher! Support has been disabled.
```

**Comment by EtherealOnyx** [comment]
Hi!

Do you happen to have the log file as well? When support disables itself, the log shows an error as to why it happened. I've been trying to figure this out but this issue is very inconsistent. I'd appreciate it!

**Comment by Endgineer** [comment]
Hey! I've attached a `latest.log` file.
[latest.log](https://github.com/EtherealOnyx/Parties/files/12459616/latest.log)

**Comment by Endgineer** [comment]
Hey! So I've tested these two mods in isolation to keep things simple; here's the [latest.log](https://github.com/EtherealOnyx/Parties/files/12459650/latest.log) and 
[debug.log](https://github.com/EtherealOnyx/Parties/files/12459653/debug.log).

**Comment by EtherealOnyx** [comment]
Thank you!
I'll look into this soon.

**Comment by EtherealOnyx** [comment]
Fixed in version 1.0.2 [here.](https://legacy.curseforge.com/minecraft/mc-mods/parties/files/4758749)
Sorry for the delay!
